public/js/eventList.js

```diff
@@ -0,0 +1,36 @@
+
+document.querySelectorAll(".event .lower").forEach(function(event) {
+    event.dataset.height = event.clientHeight
+    event.style.height = 0
+})
+
+document.querySelectorAll(".event").forEach(function(event) {
+    let clicked = 0
+    event.addEventListener('click', function() {
+        clicked++
+        let lower = this.querySelector('.lower')
+        let arrow = this.querySelector(".arrowBox")
+        if(clicked % 2) {
```

Comment:
This could change to `if (clicked) {` if you implement the previous suggestion.

Reply:
5fae0f8